bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/Clip2ThingHandler.java

```diff
@@ -631,12 +631,37 @@ public void initialize() {
         }
     }
 
+    /**
+     * Update the channel state depending on new resources sent from the bridge.
+     *
+     * @param resources a collection of Resource objects containing the new state.
+     */
+    public void onResources(Collection<Resource> resources) {
+        boolean sceneActivated = false;
+        for (Resource resource : resources) {
+            boolean skipUpdate = sceneActivated && ResourceType.SCENE == resource.getType()
+                    && resource.getSceneActive().isPresent();
```

Comment:
Just to help a bit more, getSmartSceneActive() and getSceneActive() return a tri-state result..
- Optional.ofEmpty() the resource is NOT a (smart-) scene 
- Optional.of(false) the resource is a (smart-) scene and its state is inactive
- Optional.of(true) the resource is a (smart-) scene and its state is active (actually !inactive to be precise)

Reply:
> I think you need to check for `!resource.getSceneActive().orElse(true)`

Why?

Reply:
Because your current `ifPresent()` code returns true whether the next processed scene is active or inactive, whereas you only want to skip the scene if it is specifically inactive.

Reply:
That is intentional. There can't be more than one active scene (right?), so after reaching the first one, it doesn't make sense to process anything else.

Reply:
Please see my proposed code below.

Reply:
> There can't be more than one active scene (right?)

There might be two scenes activated within one SSE package.

> so after reaching the first one

We are supposed to be ignoring any IN-active messages in an SSE package, that also contains one (or more) active messages. So the suppression of the channel updates should only apply to the specifically IN-active messages.

In addition to the above, your code steps through the resources, so you may not detect a scene active until after your have received an inactive one; and thus generating the unwanted transient inactive channel state.